RoFILib/reconfig/rrt.cpp

```diff
@@ -0,0 +1,449 @@
+#include "Algorithms.h"
+#include "test/test_rrt.h"
+
+namespace Distance
+{
+    inline double reconnections(const Configuration& curr, const Configuration& goal) {
+        auto diff = curr.diff(goal);
+        return diff.reconnections().size();
+    }
+
+    inline double rotations(const Configuration& curr, const Configuration& goal) {
+        auto diff = curr.diff(goal);
+        return diff.rotations().size();
+    }
+
+    inline double diff(const Configuration& curr, const Configuration& goal) {
+        return reconnections(curr, goal) * 1000 + rotations(curr, goal);
+    }
+}
+
+inline int findFreeIndex(int index, const std::array<bool, 6>& A)
```

Comment:
Not inline, but static (we are in a cpp file) - the same applies to other functions in this file. But a much better solution is to place them in an anonymous namespace.

Reply:
I changed my mind about anonymous namespace after reading this tweet: https://twitter.com/tvaneerd/status/1248684436796903426